backend/fms_core/services/container.py

```diff
@@ -139,7 +139,12 @@ def get_or_create_container(barcode, kind=None, name=None, coordinates=None, con
                 errors.append(f"Container kind is required to create a container.")
             elif container_parent and CONTAINER_KIND_SPECS[container_parent.kind].requires_coordinates and not coordinates:
                 errors.append(f"Parent container kind {container_parent.kind} requires that you provide coordinates.")
+            elif name is None and Container.objects.filter(name=barcode).exists():
+                    errors.append(f"Container with name {barcode} already exists. Missing container name, barcode will replace container name.")
```

Comment:
Reword the end some so it makes sense in the context that the creation will not go forward ... "... already exists while using barcode as name (container name not provided)." something like that ...